code/game/objects/structures/false_walls.dm

```diff
@@ -19,70 +104,91 @@
 	can_atmos_pass = ATMOS_PASS_DENSITY
 	rad_insulation = RAD_MEDIUM_INSULATION
 	material_flags = MATERIAL_EFFECTS
+	// Complex appearance gotta use render targets
+	blocks_emissive = EMISSIVE_BLOCK_UNIQUE
+	/// The object we are using to fake being a wall
+	var/obj/effect/falsewall_floating/visuals
```

Comment:
I think this var should be set to null on Destroy()

Reply:
also fax